cmake/FindTut.cmake

```diff
@@ -0,0 +1,47 @@
+################################################################################
+#
+# \file      cmake/FindTut.cmake
+# \author    C. Junghans
+# \copyright 2016, Los Alamos National Security, LLC.
+# \brief     Find the Tut header
+# \date      Mon 05 Dec 2016 14:21:53 AM MST
+#
+################################################################################
+
+# Find Tut headers and libraries
+#
+#  Tut_FOUND        - True if Tut is found
+#  Tut_INCLUDE_DIRS - Tut include files paths
+#
+#  Set Tut_ROOT before calling find_package to a path to add an additional
+#  search path, e.g.,
+#
+#  Usage:
+#
+#  set(Tut_ROOT "/path/to/custom/tut") # prefer over system
+#  find_package(Tut)
+#  if(Tut_FOUND)
+#    include_directories(${Tut_INCLUDE_DIR})
+#  endif()
+
+find_package(PkgConfig)
+
+# If already in cache, be silent
+if (Tut_INCLUDE_DIR)
+  set (Tut_FIND_QUIETLY TRUE)
+  pkg_check_modules(PC_TUT QUIET tut)
```

Comment:
If the pkg-config file isn't found cmake will say:
```
-- Checking for module 'tut'
--   No package 'tut' found
```
even though it says:
```
-- Found Tut: /usr/include
```
below. Without the `QUIET` in line 32, it would say that every time you run cmake even if Tut was found in an earlier run. In my experience that confuses users a lot, so that helps to keep the noise level down.